app/src/features/surveys/view/survey-animals/IndividualAnimalForm.tsx

```diff
@@ -35,15 +43,21 @@ const IndividualAnimalForm = ({ getAnimalCount }: IndividualAnimalFormProps) =>
 
   return (
     <Form>
-      <Typography variant="h4">Add New Individual</Typography>
+      <Typography variant="h4">{mode === ANIMAL_FORM_MODE.ADD ? 'Add New Individual' : 'Edit Individual'}</Typography>
+      {mode === ANIMAL_FORM_MODE.EDIT && (
+        <Typography variant="body2" color={'textSecondary'}>
+          Critter ID: {critter_id}
+        </Typography>
+      )}
       <GeneralAnimalForm />
+      <CollectionUnitAnimalForm />
       <CaptureAnimalForm />
       <MortalityAnimalForm />
       <MarkingAnimalForm />
       <MeasurementAnimalForm />
       <FamilyAnimalForm />
 
-      <FormikDevDebugger custom_payload={new Critter(values)} />
+      <FormikDevDebugger custom_payload={undefined /*new Critter(values)*/} />
```

Comment:
Should we leave this dev debugger in?

Reply:
Yeah, it's useful. Removed it since calling the constructor could affect one of the form values. That's unwanted behaviour of course, corrected it now.